src/NimBLEServer.cpp

```diff
@@ -776,6 +776,21 @@ void NimBLEServer::updateConnParams(uint16_t conn_handle,
 }// updateConnParams
 
 
+/**
+ * @brief Request an update of the data packet length.
+ * * Can only be used after a connection has been established.
+ * @param [in] conn_handle The connection handle of the peer to send the request to.
+ * @param [in] tx_octets The preferred number of payload octets to use (Range 0x001B-0x00FB).
+ * @param [in] tx_time The preferred number of microseconds to use when transmitting a single packet (Range 0x0148-0x4290).
```

Comment:
Could you please add 
@details For more information about this function and data length extension see: https://punchthrough.com/maximizing-ble-throughput-part-3-data-length-extension-dle-2/ 

Also this in the client code.

Reply:
Let me know if you are happy with the new @details.

If not, let me know what you had in mind :)

Reply:
I meant to put what I had in my comment above but yours is better 😄, thanks!